tests/testthat/test-tabyl.R

```diff
@@ -296,7 +297,9 @@ test_that("NA levels get moved to the last column in the data.frame, are suppres
   expect_equal(names(y), c("down", "up", "NA_"))
   expect_equal(
     y[["NA_"]], # column c remains numeric
-    x %>% dplyr::filter(is.na(b)) %>% tabyl(c, a)
```

Comment:
It's interesting that it broke this pipeline up into multiple lines, there are others above that it left alone